js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -51,6 +86,18 @@ const ConnectedGoogleComboAccountCard = () => {
 		( editMode && hasExistingGoogleAdsAccounts ) ||
 		( ! isConnected && hasExistingGoogleAdsAccounts );
 
+	// Show the spinner if there's an account creation in progress and account should not be claimed.
+	// If we are not showing the ConnectMC screen, for e.g when we are creating the first account,
+	// then show the spinner in the Google combo card while the Ads account is being claimed.
+	const showSpinner =
+		( Boolean( creatingWhich ) && ! shouldClaimGoogleAdsAccount ) ||
+		( ! showConnectAds && finalizeAdsAccountCreation );
+
+	const showConversionMeasurementNotice =
+		( googleAdsAccount?.status === GOOGLE_ADS_ACCOUNT_STATUS.CONNECTED ||
+			googleAdsAccount?.step === 'billing' ) &&
+		! shouldClaimGoogleAdsAccount;
```

Comment:
Happy for us to remove the `useEffect` logic that only shows this when the conversion action has first been set up, but if we're using similar logic to the `GoogleAdsAccountCard` we should make it consistent, i.e.

```suggestion
	const showConversionMeasurementNotice =
		( googleAdsAccount?.status === GOOGLE_ADS_ACCOUNT_STATUS.CONNECTED ||
			googleAdsAccount?.step === 'link_merchant' ) &&
		! shouldClaimGoogleAdsAccount;
```

Also, thought it doesn't hurt things, I don't think the `! shouldClaimGoogleAdsAccount` is necessary, because in neither condition above would `shouldClaimGoogleAdsAccount` resolve to `true`.

Reply:
I've handled this in [ac3eef9](https://github.com/woocommerce/google-listings-and-ads/pull/2644/commits/ac3eef9d1113683baa76f27671c9508ddec20486) and moved this logic to a utility function to avoid inconsistent logic.